ceph.md

```diff
@@ -152,10 +184,14 @@ When a CR containing the above is created, an Ansible pod
 running on OpenShift mounts the files in the Ceph secret
 and copies them to the EDPM host using the
 [edpm_ceph_client_files](https://github.com/openstack-k8s-operators/edpm-ansible/tree/main/roles/edpm_ceph_client_files)
-Ansible role. The Nova containers then have a copy of /etc/ceph
+Ansible role. The Nova containers then have a copy of `/etc/ceph`
 that they can use to acccess the cephx key and Ceph configuration
 file.
 
+Though containers hosted on EDPM nodes use the path `/etc/ceph`
+to access the Ceph client files, they are stored in
+`/var/lib/openstack/config/ceph` on the EDPM nodes.
```

Comment:
+1 on using a different path in the host, which is consistent with what we did in TripleO [1]

[1] https://github.com/openstack/tripleo-heat-templates/blob/master/deployment/cephadm/ceph-client.yaml#L58